protos/chaincode.proto

```diff
@@ -78,10 +78,16 @@ message ChaincodeSpec {
 // TODO: Define `codePackage`.
 message ChaincodeDeploymentSpec {
 
+    enum ExecutionEnvironment {
+        DOCKER = 0;
```

Comment:
Good thought and I can see why we that would be nice...
The reason I chose DOCKER = 0 was to avoid changes (and bugs) as most uses are for DOCKER. SYSTEM is a special case, currently used only by genesis. What we are really saying is default is to deploy DOCKER.
If we change it, we should really add an option user APIs to take it as a parameter. I chose the taking the default as opposed to hardcoding DOCKER everywhere or introducing a new user option to just using default.